kratos/utilities/cpp_tests_utilities.cpp

```diff
@@ -69,8 +71,48 @@ void Create2DGeometry(
 /***********************************************************************************/
 /***********************************************************************************/
 
+void Create2DGeometryDummyElements(ModelPart& rModelPart)
+{
+    Properties::Pointer p_prop = rModelPart.HasProperties(0) ? rModelPart.pGetProperties(0) : rModelPart.CreateNewProperties(0);
+
+    using NodeType = Node<3>;
+
```

Comment:
I think that you should check in here that there are no nodes, elements, conditions and constraints. If so, remove them all.

Reply:
¿?, there are nodes, I am creating them

Reply:
OFC, I mean in the input model part, before you create these ones. Basically we need to check that the input model part is empty. If not we wipe it and throw a warning message.

Note that with this change you're giving the possibility to use this in any context, so misuses can occur. With this we prevent them.

Reply:
Okay